src/utils.ts

```diff
@@ -38,3 +40,16 @@ export function isSPDXValid(license: string): boolean {
     return false
   }
 }
+
+export function octokitClient(token = 'repo-token', required = true): Octokit {
```

Comment:
keeping this here for now but maybe worth it moving it a separate file. Any reservations using the same client instead of [this](https://github.com/actions/dependency-review-action/blob/main/src/dependency-graph.ts#L6-L9) in `dependency-graph.ts` [since retrying once is enabled by default](https://github.com/octokit/octokit.js/blob/main/README.md#constructor-options)?
![image](https://user-images.githubusercontent.com/26176338/200006733-7eca0843-7fc3-4f8a-82d8-76a70d0e7fed.png)